.github/workflows/prod-chat-CD.yml

```
@@ -46,12 +46,19 @@ jobs:
           aws ecr describe-repositories --repository-names "${{ env.ECR_APP_NAME }}" >/dev/null 2>&1 || \
           aws ecr create-repository --repository-name "${{ env.ECR_APP_NAME }}" >/dev/null
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
       - name: Build & push CHAT image
         uses: docker/build-push-action@v4
```

actionlint (1.7.8)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)